engines/wage/wage.cpp

```diff
@@ -172,7 +172,25 @@ Common::Error WageEngine::run() {
 	return Common::kNoError;
 }
 
+// Resetting required variables
+void WageEngine::resetAllParameters() {
```

Comment:
Maybe name it as `resetState()`? The term "parameters" is not used in this engine.